_data/simple-icons.json

```
@@ -10472,9 +10472,9 @@
         },
         {
             "title": "Node.js",
-            "hex": "339933",
-            "source": "https://nodejs.org/en/about/resources/",
-            "guidelines": "https://nodejs.org/en/about/resources/"
+            "hex": "5FA04E",
+            "source": "https://github.com/nodejs/nodejs.org/blob/941b0022c38739d6a9277d558f6b9c78e13a4262/public/static/logos/jsIconGreen.svg",
+            "guidelines": "https://nodejs.org/en/about/branding"
```

```suggestion
            "source": "https://nodejs.org/en/about/branding",
            "guidelines": "https://nodejs.org/en/about/branding"
```

Since both the trademark policy and the logos are listed on this page this is what I would go with.